src/commands/mod.rs

```diff
@@ -486,13 +486,20 @@ impl DaemonControl {
         // the spend from a set of optional candidates.
         // Otherwise, only the specified coins will be used, all as mandatory candidates.
         let candidate_coins: Vec<CandidateCoin> = if coins_outpoints.is_empty() {
-            // We only select confirmed coins for now. Including unconfirmed ones as well would
-            // introduce a whole bunch of additional complexity.
+            // We only select confirmed coins and unconfirmed change for now.
+            // Including unconfirmed external deposits would introduce a whole bunch
+            // of additional complexity.
```

Comment:
It's more that it would be "unsafe" without being explicitly acknowledged by the user, as these coins may disappear at any time.